command_wrapper.gs

```diff
@@ -345,8 +445,8 @@ class ListActiveCommand extends Command {
     this.rows = this.tracking_sheet.getAllRows();
   }
   
-  formulateUserResponse(){
-    var message_out = 'These are all the active requests in this channel:\n';
+  notify(){
+    var message_out = listHeaderMessage('listactive');
 
     var classScope = this;
```

Comment:
No need to change it now, but by using arrow functions you can avoid having to explicitly track the scope.

https://jeffreyeverhart.com/2020/02/12/es6-features-for-google-apps-script-arrow-functions/

Reply:
done in latest commit